depobs/worker/tasks.py

```diff
@@ -373,3 +375,53 @@ def fetch_and_save_registry_entries(package_names: Iterable[str]) -> List[Dict]:
         )
     )
     return npm_registry_entries
+
+def get_github_advisories(package_name: str) -> List[Dict]:
+
+    github_client = current_app.config["GITHUB_CLIENT"]
+    base_url = github_client["base_url"]
+    github_auth_token = github_client["github_auth_token"]
+
+    headers = {"Authorization": "token " + github_auth_token}
+
+    query = (
+    """
+    {
+        securityVulnerabilities(ecosystem: NPM, first: 100, package: \""""
+        + package_name
+        + """\", orderBy: {field: UPDATED_AT, direction: DESC}) {
+            nodes {
+                advisory {
+                    id, description, permalink, publishedAt, severity, summary, updatedAt, withdrawnAt
+                }
+                package {
+                    name
+                }
+            }
+            pageInfo {
+                endCursor, hasNextPage, hasPreviousPage, startCursor
+            }
+            totalCount
+        }
+    }
+    """
+    )
+
+    response = json.loads(
```

Comment:
nit: call `response.raise_for_status()` https://requests.readthedocs.io/en/latest/api/#requests.Response.raise_for_status so we don't try to parse or save errors then use `response.json()` https://requests.readthedocs.io/en/latest/user/quickstart/#json-response-content